### ISSUE TITLE
[Bug] Webtoon: Problem with searching and Page list empty error

### ISSUE THREAD

**Issue by psychopomp9** [inorichi/tachiyomi-extensions#3673]
### Device information
- Tachiyomi version: 0.9.2 stable
- Android version: 9

### Source information
- Name: Webtoon
- Extension version: 1.2.16

### Expected behavior
When searching for manga in webtoon source it shows an error even though the manga is there on the source!
When trying to read some mangas from webtoon source it shows an error 'page list is empty'. i.e. shriek 

### Actual behavior
It should have searched for the manga and show the results.
Chapters should have been loaded rather than showing an error.

### Other details
I tried searching a few mangas in webtoon source, i.e parallel city and it shows an error message. When trying to global search the manga doesn't show up from webtoon source.

**Comment by AwesomeQuest** [comment]
I have the same issue but I can read manga just fine

**Comment by Ha5yA** [comment]
Same issue with searching... but webtoons already in library can be read. Also if searched in web view the webtoon can be found.
But when tried to search within extension it doesn't work

**Comment by psychopomp9** [comment]
If you don't mind, can you check if 'chiller' is giving a page list empty error or not on webtoon source? If you can search it on the source though!

**Comment by Ha5yA** [comment]
> 
> 
> If you don't mind, can you check if 'chiller' is giving a page list empty error or not on webtoon source? If you can search it on the source though!

It shows "No Results found" no matter anything i type

**Comment by AwesomeQuest** [comment]
> If you don't mind, can you check if 'chiller' is giving a page list empty error or not on webtoon source? If you can search it on the source though!

'chiller' does not return any results.

**Comment by psychopomp9** [comment]
Search in webtoon is broke! That's confirmed. And showing page list empty for some mangas was happening with 'chiller' webtoon. But we can confirm it only after the problem with search is resolved..

**Comment by psychopomp9** [comment]
![Screenshot_2020-07-03-10-37-07-263_eu kanade tachiyomi](https://user-images.githubusercontent.com/67730075/86433768-39379f00-bd19-11ea-8ced-0af0d51a52f7.jpg)

Finally find 'chiller' after scrolling through whole webtoon database! And it shows page list is empty error!.

**Comment by SnakeDoc83** [comment]
Search will be fixed with the next update, but not the "page list empty" error. When I look up "Chiller" or "Shriek" on their website the chapters aren't readable. I'm guessing you need to be logged in to an account to view those and possibly some other webtoons.

**Comment by psychopomp9** [comment]
![Screenshot (11)](https://user-images.githubusercontent.com/67730075/86436851-bdd9eb80-bd20-11ea-8785-bcd65089edbb.png)
it's working fine for me without logging in on website! @SnakeDoc83

**Comment by SnakeDoc83** [comment]
Ugh, Privacy Badger blocked the images on my PC.

**Comment by psychopomp9** [comment]
it happens. 'shriek' wasn't loading on the website a few days ago(with/without logging in), I reported the issue to the line corp. and now it's working, but they didn't fix it! it's working on its own now without a fix!
And will the page list empty error be fixed in the next update or should I create a new thread for that?

**Comment by SnakeDoc83** [comment]
If you don't mind, open a separate issue.
